### DISCUSSION TITLE
Is the L2 distance score in vector search an inverted score?

### DISCUSSION THREAD

**Discussion by yy0vnne** [lancedb/lancedb#1261] [Q&A] [answered]
Hi there, the L2 scores and cosine scores for LanceDB vector search suggests that smaller ```_distance``` scores means more similar.
But the general understanding is that for L2 scores, smaller ```_distance``` means more similar, while for cosine score, greater ```_distance``` means more similar. May I check if L2 distance is an inverted score in LanceDB?

**Comment by westonpace** [comment, marked as answer]
The L2 score is not inverted.

The cosine score is "cosine distance" and not "cosine similarity".

Cosine distance is defined as `1 - cosine_similarity` (cosine distance is inverted cosine similarity).

In all cases a smaller `_distance` means more similar.
